config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<!-- This is a checkstyle configuration file. For descriptions of
```

Comment:
Do we actually pass all these checks already? Are there a lot of warnings? I'd rather not start off with too many broken windows.